packages/table/src/table.tsx

```diff
@@ -403,6 +403,8 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         [QuadrantType.TOP_LEFT]: this.generateQuadrantRefHandlers(QuadrantType.TOP_LEFT),
     };
 
+    private wasMainQuadrantScrollChangedFromOtherOnWheelCallback = false;
```

Comment:
Oh dear gods, that sounds like _I_ came up with that name. 

Can we do better here?